apps/dashboard/src/core-ui/sidebar/connect.tsx

```diff
@@ -13,16 +13,16 @@ type ConnectSidebarProps = {
 };
 
 const links: Route[] = [
-  {
-    path: "/dashboard/connect/in-app-wallets",
-    title: "In-App Wallets",
-    name: "embedded-wallets",
-  },
   {
     path: "/dashboard/connect/analytics",
     title: "Analytics",
     name: "analytics",
   },
```

Comment:
By default when I click connect tab, it still takes me to in app wallet, can you change it so it defaults to analytics?

Also I think we can get a quick win by updating the empty state here.

Maybe say something like "integrate the connect SDK in your app to view analytics." + Link to docs for each platform

Reply:
Done, screenshot in the PR description

Reply:
Nice that's perfect. 

Nit: can you update the sdk  cards content to be left aligned? Cleaner imo

Reply:
we'll re-visit this as part of teams migration this is functional lets ship it